<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoEditor by Radu Szasz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>441</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>549</width>
       <height>439</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuAdjust">
     <property name="title">
      <string>Adjust</string>
     </property>
     <addaction name="actionBrightness"/>
     <addaction name="actionContrast"/>
     <addaction name="actionHue"/>
     <addaction name="actionSaturation"/>
    </widget>
    <addaction name="actionResize"/>
    <addaction name="menuAdjust"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionBlack_White"/>
    <addaction name="actionBlur"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuAlignment">
     <property name="title">
      <string>Alignment</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionCentered"/>
    </widget>
    <addaction name="actionFit_into_screen"/>
    <addaction name="actionFull_image"/>
    <addaction name="menuAlignment"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuFilter"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionHue">
   <property name="text">
    <string>Hue</string>
   </property>
  </action>
  <action name="actionSaturation">
   <property name="text">
    <string>Saturation</string>
   </property>
  </action>
  <action name="actionBlack_White">
   <property name="text">
    <string>Black/White</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSharpness">
   <property name="text">
    <string>Sharpness</string>
   </property>
  </action>
  <action name="actionFit_into_screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit into screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFull_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full image</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCentered">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Centered</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
